libsrc/target/sms/sms.hdr

```diff
@@ -2,3 +2,13 @@
 	DEFC	SpriteSet          = 0       ; 0 for sprites to use tiles 0-255, 1 for 256+
 	DEFC	NameTableAddress   = $3800   ; must be a multiple of $800; usually $3800; fills $700 bytes (unstretched)
 	DEFC	SpriteTableAddress = $3f00   ; must be a multiple of $100; usually $3f00; fills $100 bytes
+
+
+	DEFC	__IO_VDP_DATA      = 0xbe
```

Comment:
Defines for accessing the VDP registers. Maybe these should be pulled from the video library? They seem to be defined in multiple places and I wasn't sure which ones to use so here they are again.

Reply:
...so one of my plans for a reorganisation is to put all the defines needed for a target library into a single place in `libsrc/target/XXX/?????.inc` - for example the ports for the PSG, 1 bit sound, graphics resolution/flags etc

This would remove the duplicated definitions. I did start moving stuff around in preparation but as usual with these things I ended up with a somewhat larger snowball than I intended to.